src/databaseTest/java/net/sf/jabref/shared/TestManager.java

```diff
@@ -5,6 +5,10 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * {@link TestManager} provides helping methods for database tests.
```

Comment:
I would either say "This class provides.." or use `@code` annotation because linking to the same javadoc doesn't make much sense in my opinion :smile: The pull request is still good from my point of view.